_data/projects/30SecondsOfC++.yml

```diff
@@ -0,0 +1,11 @@
+name: 30 Seconds of C++
+desc: A collection of STL features (functions/libraries) of C++ which can be learned in 30 seconds or less.
+site: https://github.com/Bhupesh-V/30-seconds-of-cpp
+tags:
+- c++
+- c++11
+- c++14
+- c++17
+upforgrabs:
+  name: 30-Seconds-of-C++
```

Comment:
@Bhupesh-V This should be the link to your label, such as "good first issue", "up-for-grabs" etc. and also the following link should be for that only. Please change